documentation_builder/simulating.ipynb

```diff
@@ -758,13 +759,14 @@
       "PRODUCING REACTIONS -- Pyruvate\n",
       "-------------------------------\n",
       "  %             FLUX   RXN ID                        REACTION                       \n",
-      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
-      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
+      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
```

Comment:
No worries about all the comments. It's all relevant to the discussion of
this bug.
On Mar 16, 2016 1:15 PM, "Christian Diener" <notifications@github.com>
wrote:

> In documentation_builder/simulating.ipynb
> <https://github.com/opencobra/cobrapy/pull/239#discussion_r56406338>:
>
> > @@ -758,13 +759,14 @@
> >        "PRODUCING REACTIONS -- Pyruvate\n",
> >        "-------------------------------\n",
> >        "  %             FLUX   RXN ID                        REACTION                       \n",
> > -      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
> > -      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
> > +      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
>
> @pstjohn <https://github.com/pstjohn> yes, completely agree. Sorry for
> spamming comments here. Maybe we should open a gitter to discuss stuff like
> this...
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencobra/cobrapy/pull/239/files/8a60f16e28ddb40995a91b00b3b74e188f8eb93b#r56406338>
>